package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "private": true,
+  "author": "18F",
+  "license": "CC0-1.0",
+  "scripts": {
+    "postinstall": "bundle",
```

Comment:
This may do some weird stuff on Federalist. You'll be calling bundle twice since Federalist does that for you. Probably not a huge cause for concern, but wanted to point out.

Reply:
Is this file even needed since all of the assets should be coming from jekyll theme gem now?

Reply:
Thanks for flagging this @jmhooper and that's a great point @hursey013. I'm going to remove this .json file